eng/pipelines/templates/stages/archetype-sdk-publish-net.yml

```diff
@@ -0,0 +1,202 @@
+            # version and a basic override of $Language should be supported before we can remove this build-tools repo script.
+            filePath: '$(BuildToolScripts)/create-tags-and-git-release.ps1'
+            arguments: '-artifactLocation $(Artifacts) -workingDirectory $(System.DefaultWorkingDirectory) -packageRepository Nuget -releaseSha $(Build.SourceVersion) -repoId $(Build.Repository.Id)'
+          env:
+            GH_TOKEN: $(azuresdk-github-pat)
+
+        - pwsh: |
+            Write-Host "##vso[task.setvariable variable=releaseVar;isOutput=true]$(release)"
+          displayName: 'Set Release Variable'
+          name: SetRelease
+
+  - deployment: Publish_Package
+    dependsOn: SignPackage
+    environment: net-release
+    pool:
+      name: ${{ parameters.SigningPool.name }}
+      ${{ if eq(parameters.OSName, 'macOS') }}:
+        vmImage: ${{ parameters.SigningPool.image }}
+      ${{ else }}:
+        image: ${{ parameters.SigningPool.image }}
+      os: ${{ parameters.SigningPool.os }}
+
+    templateContext:
+      type: releaseJob
+      isProduction: false
+    strategy:
+      runOnce:
+        deploy:
+          steps:
+            - ${{ if eq(parameters.ShouldSign, 'true') }}:
+              - task: 1ES.DownloadPipelineArtifact@1
+                displayName: Download Signed Packages
+                inputs:
+                  ${{ if eq(parameters.ShouldSign, 'true') }}:
+                    artifactName: packages-signed
+                    targetPath: $(Artifacts)
+                  ${{ else }}:
+                    artifactName: packages
+                    targetPath: $(Artifacts)
+
+            - ${{ if eq(parameters.ShouldPublish, 'true') }}:
+              - task: 1ES.PublishNuget@1
+                displayName: Publish to DevOps Feed
+                condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/main')) # Optional condition
+                inputs:
+                  useDotNetTask: false
+                  packagesToPush: '$(Artifacts)/*.nupkg;!$(Artifacts)/*.symbols.nupkg'
+                  packageParentPath: '$(Artifacts)'
+                  publishVstsFeed: ${{ parameters.DevOpsFeedID }}
+                  nuGetFeedType: internal
+
+  - job: PostPackagePublish
+    dependsOn:
+     - Publish_Package
+     - SignPackage
+    displayName: Post Package Publish
+    variables:
+      - name: release
+        value: $[ dependencies.SignPackage.outputs['SetRelease.releaseVar'] ]
+    pool:
+      name: ${{ parameters.SigningPool.name }}
+      ${{ if eq(parameters.OSName, 'macOS') }}:
+        vmImage: ${{ parameters.SigningPool.image }}
+      ${{ else }}:
+        image: ${{ parameters.SigningPool.image }}
+      os: ${{ parameters.SigningPool.os }}
+    steps:
+      - checkout: azure-sdk-build-tools
+        path: _azure-sdk-build-tools
```

Comment:
So we need 2 copies?

Reply:
There is a single checkout for azure-sdk-build-tools, and one for azure-sdk-tools, which is `self`. I'm confused at the implication.

Reply:
Turns ou `publish-symbols` does need a local azure-sdk-build-tools script. So still leaving it, but left a comment as to why.